### PR TITLE
feat: make test run id available as $testId template variable

### PR DESCRIPTION
**Pull request by hassy**
This PR makes the test run ID of the current test run available as `$testId` template variable. The variable can be used in:

- `config` section as `{{ $testId }}`
- in scenarios as `{{ $testId }}`
- custom JS code as `vuContext.vars.$testId`

Works locally, and in cloud tests (AWS Lambda and AWS Fargate).

### CODE REVIEW

File: packages/artillery/lib/cmds/run.js [hunk 1]
```diff
@@ -143,6 +145,10 @@ RunCommand.runCommandImplementation = async function (flags, argv, args) {
   }
 
   try {
+    const testRunId = process.env.ARTILLERY_TEST_RUN_ID || generateId('t');
```
Comment (hassy) [inline packages/artillery/lib/cmds/run.js:148]: Pick up override from the environment - this is for cloud workers only.

File: packages/artillery/lib/platform/local/artillery-worker-local.js [hunk 2]
```diff
@@ -106,7 +106,7 @@ class ArtilleryWorker {
     const { script, payload, options } = opts;
     this.worker.postMessage({
       command: 'prepare',
-      opts: { script, payload, options }
+      opts: { script, payload, options, testRunId: global.artillery.testRunId }
```
Comment (hassy) [inline packages/artillery/lib/platform/local/artillery-worker-local.js:109]: Pass test run ID from main thread to worker threads.

File: packages/core/lib/runner.js [hunk 3]
```diff
@@ -438,7 +438,8 @@ function createContext(script, contextVars, additionalProperties = {}) {
         target: script.config.target,
         $environment: script._environment,
         $processEnvironment: process.env, // TODO: deprecate
-        $env: process.env
+        $env: process.env,
+        $testId: global.artillery.testRunId,
```
Comment (hassy) [inline packages/core/lib/runner.js:442]: Makes the variable available in scenarios.

**Review by bernardobridge — APPROVED** [review]
🚢

**Pull request merged**